### PR TITLE
docs(process): coc review notes

### PR DESCRIPTION
**Pull request by lirantal**
Added further emphasis on communication of sec wg members

### CODE REVIEW

File: processes/security_team_onboarding.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Patiently wait for feedback and +1’s from the team and a reasonable time to en
 Once acceptance has been acquired by group members, the following should take place to setup the new member:
 
 * Confirm that the new member has read, understands, and agrees to uphold the
-  [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md).
+  [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md). The work of triaging vulnerabilities at the Security WG is about handling sensitive matters with open source maintainers and communication should be handled with care. If in doubt on how to communicate reach out to the Moderation WG team and ask for guidance. As a member of the Security WG please also be aware of public statements your make and interactions.
```
Comment (benjamingr) [inline processes/security_team_onboarding.md]: Moderation is a team not a wg.

**Comment by benjamingr** [comment]
This LGTM :]

File: processes/security_team_onboarding.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Patiently wait for feedback and +1’s from the team and a reasonable time to en
...
 * Mentor should ping @jasnell, @gibfahn, @cjihrig or @mhdawson to add the new member to the repository.
```
Comment (cjihrig) [inline processes/security_team_onboarding.md]: Unrelated to this PR, but this list should likely be updated. Based on the current membership list, I would drop James and Gibson, and add @sam-github and @ChALkeR (assuming they're OK with it).

File: processes/security_team_onboarding.md (hunk 1 above)
Comment (cjihrig) [inline processes/security_team_onboarding.md]:
```suggestion
  [Code of Conduct](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md). The work of triaging vulnerabilities in the Security WG is about handling sensitive matters with open source maintainers, and communication should be handled with care. If in doubt on how to communicate reach out to the Moderation team and ask for guidance. As a member of the Security WG please also be aware of public statements you make.
```

File: processes/security_team_onboarding.md (hunk 2 above)
Comment (mhdawson) [reply processes/security_team_onboarding.md]: +1 from me

File: processes/security_team_onboarding.md (hunk 1 above)
Comment (mhdawson) [reply processes/security_team_onboarding.md]: Pushed commit to fix that.  @lirantal hope that's ok.

Comment (lirantal) [reply processes/security_team_onboarding.md]: of course it is! thank you both ;-)

File: processes/security_team_onboarding.md (hunk 2 above)
Comment (lirantal) [reply processes/security_team_onboarding.md]: please confirm @sam-github and @ChALkeR

File: processes/security_team_onboarding.md (hunk 1 above)
Comment (sam-github) [inline processes/security_team_onboarding.md]: The moderation group can help, but I  think asking on the security-wg slack is a good first step, too. Or technology aside, its good to work together with other people on the sec-wg before going the next step and involving moderation team.

Comment (lirantal) [reply processes/security_team_onboarding.md]: Understand where you're coming from and hopefully it should go without saying that discussing it with friends on the wg slack is a good first step. With that said, in the matters of communicating on public forums the sec wg may not be well trained on it even if they have good intentions - therefore outlining here that the Moderation team is a helpful channel is important to mention in my opinion.

File: processes/security_team_onboarding.md (hunk 2 above)
Comment (lirantal) [reply processes/security_team_onboarding.md]: I'll open a follow-up PR for this

File: processes/security_team_onboarding.md (hunk 1 above)
Comment (lirantal) [reply processes/security_team_onboarding.md]: Will resolve the conversation and we can pick it up in a new PR if needed

**Pull request merged**